common/lib/xmodule/xmodule/static_content.py

```diff
@@ -186,6 +186,9 @@ def main():
     Generate
     Usage: static_content.py <output_root>
     """
+    from django.conf import settings
+    settings.configure()
```

Comment:
@nedbat Shouldn't we be passing the `settings` to `configure` too, just like we are for the other commands? 
https://github.com/edx/edx-platform/blob/master/pavelib/assets.py#L332-L341

https://docs.djangoproject.com/en/1.8/topics/settings/#using-settings-without-setting-django-settings-module